roles/nova-common/meta/main.yml

```diff
@@ -25,3 +25,5 @@ dependencies:
     rootwrap: True
     alternatives: "{{ nova.alternatives }}"
     when: openstack_install_method == 'package'
+  - role: ceph-common
+    when: ceph.enabled |bool
```

Comment:
This needs some better conditional, so that it's only added when ceph for VMs is enabled, rather than the generic cep.enabled.